### DISCUSSION TITLE
Android Auto UX

### DISCUSSION THREAD

**Discussion by musshorn** [osmandapp/OsmAnd#17784] [General]
I've just started using the latest release of OsmAnd+ (4.5.8) and I feel like the Android Auto UX has regressed somewhat? . Previously the search button was visible immediately but now it's hidden behind opening the fast menu then hitting the magnifying glass icon. Do you think it would be better to keep search as a one press event? And should it be listed as a item in the new menu?

**Comment by dmpr0** [comment]
Hello! 
Search button should be available right after start in the right top corner.
<img src="https://github.com/osmandapp/OsmAnd/assets/9365308/e738cee8-2260-4e4d-8396-f2e5c9288372"  width="600" height="auto"/>

**Reply by musshorn** [reply]
Ahh ok, do you think that's kind of hidden though? So if you were new user to the app would having it in the menu on the left would be more intuitive?

**Comment by ghost** [comment]
It seems to me that the menu on the left shouldn't pop up on start. Instead it should just go to "Free Ride" mode showing the map. On the rare times I want a route, I'll bring up the menu. But most drives I know where I'm going, the only reason OsmAnd is displayed is because Android Auto always defaults to bringing up a Nav app.

On a side note, there shouldn't be a > on the Free Ride option. There isn't a submenu on it.

**Reply by musshorn** [reply]
I do agree with that, having to select free ride repeatedly gets annoying. Especially if the screen is interrupted (eg, reversing camera comes on). 

My use case seems to be different, I've been travelling extensively and having search easily accessible was useful for navigation (or working out if mapping need to be done!) so I think I'm probably more sensitive to having it slightly harder to find

**Comment by dmpr0** [comment]
I think it not possible to hide the landing menu on the start, we use default layout and behavior providing by Android Auto.

**Reply by dbareis** [reply]
Then the landing menu doesn't need to be used, or can be empty, or conditional of user settings?

**Reply by dmpr0** [reply]
It has **Free ride** button, to hide it

**Reply by dbareis** [reply]
@dmpr0 Yes and I originally wondered if I had downloaded a trial app because of that, why not call it "Exit" or simply have an "X" to close, and probably auto close after 10 or so seconds if you are insisting it be shown.

**Reply by dmpr0** [reply]
Android Auto has limited layout for Navigation apps. 
Android Auto has limitation for page limit. 
We don't have plans to change it any time soon.

**Reply by dbareis** [reply]
It also doesn't support pinch to zoom or choosing destinations on the map.  I will always be using it to car navigation so I will look for another app with better AA support.  I might check back in a year or two to see if there are any improvements.  I really hate how search is only available from the main page also.
